docs/src/pages/components/badges/badges.md

```diff
@@ -66,3 +66,12 @@ import BadgeUnstyled from '@material-ui/unstyled/BadgeUnstyled';
 ```
 
 {{"demo": "pages/components/badges/UnstyledBadge.js"}}
+
+## Accessibility
+
+- `Badges` can be given accessible label using `aria-label` or `aria-labelledby`.
+- You can provide description using `aria-describedby`.
+
+{{"demo": "pages/components/badges/AccessibleBadges.js"}}
+
+- If you are using `Badges` with `[role = "status"]` then connect other components properly by setting correct `id`, `aria-controls` and `aria-live`. You can get more info [here](https://www.w3.org/TR/wai-aria-1.1/#status).
```

Comment:
That is too "hand-wavy". Seems like we rather want a dedicated demo for this pattern.

Reply:
Will this serve as a good demo?
```
import React, {useState} from "react";
import Badge from "@material-ui/core/Badge";

export default function Counter() {

	const [count, setCount] = useState(1);

	return (
    		<>
      			<div style={{display: "inline-block", margin:"13px"}}>
				<Badge role="status" id="badge1" color="primary" badgeContent={count} style={{marginRight:"22px"}}> </Badge>
  				<button onClick={()=>setCount(count+1)} type="button" id="btn1" aria-controls="badge1">Increment</button>
      			</div>
    		</>
               );
}
```